components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.operation-bar/public/js/operation-bar.js

```diff
@@ -42,14 +51,31 @@ function submitForm(formId) {
             uriencodedQueryStr += prefix + input.attr("id") + "=" + input.val();
         } else if (input.data("param-type") == "form") {
             var prefix = (uriencodedFormStr == "") ? "" : "&";
-            uriencodedFormStr += prefix + input.attr("id") + "=" + input.val();
-            //payload[input.attr("id")] = input.val();
+            if (input.attr("type") == "checkbox" || input.attr("type") == "radio"){
+
+                if (isItemSelected == undefined){
+                    isItemSelected = false;
+                }
+                if (input.is(':checked')){
+                    isItemSelected = true;
+                    uriencodedFormStr += prefix + input.attr("name") + "=" + input.val();
+                }
+            }else{
+                uriencodedFormStr += prefix + input.attr("id") + "=" + input.val();
+            }
         }
     });
+
+    if (isItemSelected === false){
+        title.html("Please Select One Option");
+        statusIcon.attr("class", defaultStatusClasses + " fw-error");
+        $(modalPopupContent).html(content.html());
+        return false;
+    }
+
```

Comment:
Please format the above block

Reply:
This is fixed 0ce103e1ad33617f640d5c03b57477f2b5adb4db